src/integration_tests/local_tests.py

```diff
@@ -70,6 +75,53 @@ def client(database, storage):
     yield client
 
 
+def test_dataset(client):
+    with open("../test_data/dataset.json") as f:
```

Comment:
Doc string missing and there is one print left in the code

Reply:
Done